.gitignore

```diff
@@ -11,4 +11,7 @@ doc/*
 bin/*
 !bin/recurly
 
+#Gems should not checkin lock files
+Gemfile.lock
```

Comment:
What do you think about referencing the ykatz article here in the comment so we can remember our reasoning later on?

http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/